streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionJobVertex.java

```diff
@@ -0,0 +1,174 @@
+package org.ray.streaming.runtime.core.graph.executiongraph;
+
+import com.google.common.base.Preconditions;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.aeonbits.owner.ConfigCache;
+import org.ray.api.RayActor;
+import org.ray.streaming.jobgraph.JobVertex;
+import org.ray.streaming.jobgraph.LanguageType;
+import org.ray.streaming.jobgraph.VertexType;
+import org.ray.streaming.operator.StreamOperator;
+import org.ray.streaming.runtime.config.master.ResourceConfig;
+import org.ray.streaming.runtime.core.processor.ProcessBuilder;
+import org.ray.streaming.runtime.core.processor.StreamProcessor;
+import org.ray.streaming.runtime.worker.JobWorker;
+
+/**
+ * Physical job vertex which including parallelism execution vertex.
+ */
+public class ExecutionJobVertex {
+
+  private final String jobName;
+  private final int jobVertexId;
+  private final String jobVertexName;
+  private final JobVertex jobVertex;
+  private final StreamProcessor streamProcessor;
+  private int parallelism;
+  private Map<String, String> jobConfig;
+  private Map<String, Double> resources;
+  private long buildTime;
```

Comment:
What's this for? `ExecutionGraph` already has  a `buildTime`. Spreading same informations into multi places incurs code maintenance cost, can be inconsistent easily, and doesn't conform to `The Single Responsibility Principle`. It's the responsibility of `ExecutionGraph` to maintain these information. Same for `jobName`, `jobConfig`

Reply:
Done simplify.